MediaRecorder.bs

```diff
@@ -531,6 +543,7 @@ dictionary MediaRecorderOptions {
   unsigned long videoBitsPerSecond;
   unsigned long bitsPerSecond;
   BitrateMode audioBitrateMode = "variable";
+  DOMHighResTimeStamp videoKeyFrameInterval;
```

Comment:
[`DOMHighResTimestamp`](https://www.w3.org/TR/hr-time/#sec-domhighrestimestamp) is documented to be able to express a duration. Double seems worse because it can be negative or NaN and behavior re that need to be defined. 

Since the type is defined, is there any reason to say it's seconds? It's covered by the `DOMHighResTimestamp` spec.

Reply:
DOMHighResTimestamp is just a typedef for a double. The reason the this type can be used to also describe a time interval is because you can say "interval = timestamp2 - timestamp1" without type conversion.

I have never seen DOMHighResTimestamp used to express an interval when it is an attribute, but getStats() for example is full of uses of double for intervals in seconds.

Reply:
Although I see the appeal of ms instead of s, was just thinking of mathcing with other APIs, but if this is used as intervals in other places then no objection from me

Reply:
performance.now() is probably the most common usage of HiResTimeStamp, and it's an interval (measured from the time origin of the window or worker).
so it's not novel to use it for intervals.

Reply:
Ack